lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -1684,6 +1685,12 @@ def main():
         ],
     )
 
+    if module.params.get('group') and module.params.get('group_id'):
+        module.deprecate(
+            msg='Support for passing both group and group_id has been deprecated. '
+            'Currently group_id is ignored, in future passing both will result in an error.',
```

Comment:
```suggestion
            'Currently group_id is ignored, in future passing both will result in an error',
```
`deprecate()` is adding a period at the end of `msg`, so if you add one here you will get two :)